sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md

```diff
@@ -24,7 +24,10 @@ Add-SPOSiteCollectionAppCatalog -Site <SpoSitePipeBind> [-WhatIf] [-Confirm] [<C
 
 ## DESCRIPTION
 
-Use this cmdlet to add a site collection scoped app catalog to the specified site.
+Use this cmdlet to add a site collection scoped app catalog to the specified site.To run this cmdle user must have the following roles:
+* SharePoint tenant administrator
```

Comment:
This should be 'SharePoint Admin role or Global Administrator role'.

Reply:
agreed, Thanks